tests/utils-shared/pom.xml

```diff
@@ -49,5 +49,9 @@
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+        </dependency>
```

Comment:
This is not possible - as it will add JUnit 5 to the old testsuite
```suggestion

```

Reply:
ok